File: SoloLearn/Snap, Crackle and Pop/SnapCracklePop.c
#include <stdio.h>

int main() {
    int number;
    for (int i=0; i<6; i++) {
        scanf("%d", &number);
        if (number % 3 == 0) 
            printf("Pop ");
        else if (number % 2 == 0)
            printf("Crackle ");
        else
            printf("Snap ");
    }
    return 0;
} 

File: SoloLearn/Snap, Crackle and Pop/SnapCracklePop.swift
for i in 1...6 {
    if let number = readLine(), let n = Int(number) {
        if n % 3 == 0 {
            print("Pop", terminator: " ")
        } else if n % 2 == 0 {
            print("Crackle", terminator: " ")
        } else {
            print("Snap", terminator: " ")
        }
    }
}